src/current/v23.1/create-changefeed.md

```diff
@@ -335,6 +335,19 @@ CREATE CHANGEFEED FOR TABLE table_name INTO 'external://kafka_sink'
   WITH resolved;
 ~~~
 
+### Disallow schema changes on tables to improve changefeed performance
+
+{% include_cached new-in.html version="v23.1.2" %} Use the `schema_locked` [storage parameter]({% link {{ page.version.version }}/with-storage-parameter.md %}) to disallow [schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}) on a watched table, which helps to decrease the latency between a write committing to a table and it emitting to the [changefeed's sink]({% link {{ page.version.version }}/changefeed-sinks.md %}). You can lock the table before creating a changefeed or while a changefeed is running, which will enable the performance improvement for changefeeds watching the particular table.
+
+Enable `schema_locked` on the watched table with the [`ALTER TABLE`]({% link {{ page.version.version }}/alter-table.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE watched_table SET (schema_locked = true);
+~~~
+
+While `schema_locked` is enabled on a table, attempted schema changes on the table will be rejected and an error returned. If you need to run a schema change on the locked table, unlock the table with `schema_locked = false`, complete the schema change, and then lock the table again with `schema_locked = true`. The changefeed will run as normal while `schema_locked = false`, but it will not benefit from the performance optimization.
```

Comment:
Maybe we should also include some example syntax for how to reset the field. They can do that by doing either:
```
ALTER TABLE watched_table SET (schema_locked = false);
```
or
```
ALTER TABLE watched_table RESET (schema_locked);
```

Ditto for all the other versions.

Reply:
Done!